From 0131a93c7d95f8d27d79463750a1fdf471ba79b1 Mon Sep 17 00:00:00 2001
From: "Rick Farina (Zero_Chaos)" <zerochaos@gentoo.org>
Date: Fri, 19 Oct 2012 23:27:03 -0400
Subject: [PATCH] prevent cleaning .a files

The releng specs already remove all the static libs from the system
We do not need to do it with catalyst as this breaks all other livecds

Signed-off-by: Rick Farina (Zero_Chaos) <zerochaos@gentoo.org>
---
 targets/livecd-stage2/livecd-stage2-controller.sh |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/targets/livecd-stage2/livecd-stage2-controller.sh b/targets/livecd-stage2/livecd-stage2-controller.sh
index 687c208..57dcff7 100644
--- a/targets/livecd-stage2/livecd-stage2-controller.sh
+++ b/targets/livecd-stage2/livecd-stage2-controller.sh
@@ -109,13 +109,6 @@ case $1 in
 			# Zap all .pyc and .pyo files
 			find ${clst_chroot_path}/usr/lib* -iname "*.py[co]" -exec rm -f {} \;
 		fi
-		# Cleanup all .a files except libgcc.a, *_nonshared.a and
-		# /usr/lib/portage/bin/*.a
-		find ${clst_chroot_path}usr/lib* -type f -iname "*.a" | \
-			grep -v 'libgcc.a' | \
-			grep -v 'nonshared.a' | \
-			grep -v '/usr/lib/portage/bin/' | \
-			grep -v 'libgcc_eh.a' | xargs rm -f
 		rm -f ${clst_chroot_path}/tmp/packages.txt
 		;;
 
-- 
1.7.8.6

